website/pages/en/index.js

```diff
@@ -138,17 +138,20 @@ class Index extends React.Component {
                 <div className="blockImage">
                   <div>
                     <MarkdownBlock>
-                      {typeAnnotationExample}
+                      {asyncExample}
                     </MarkdownBlock>
                   </div>
                 </div>
                 <div className="blockContent">
                   <h2>
-                    Type Annotations
+                    Async
                   </h2>
                   <div>
                     <MarkdownBlock>
-                      Type annotations allow for code to be explicitly typed on parameters, class member variables and return values.
+                      **[Asynchronous operations](https://docs.hhvm.com/hack/asynchronous-operations/introduction)** allow _cooperative multi-tasking_. Code that utilizes the Async Infra can hide I/O latency and data fetching, minimizing the downtime a program has to be stalled for I/O or similar operations.
+                    </MarkdownBlock>
+                    <MarkdownBlock>
+                      **Async is not multithreading**. HHVM executes a program's code in one main request thread.
```

Comment:
I don't think limitation details like this belong on the home page. We've already said it's cooperative multi-tasking.